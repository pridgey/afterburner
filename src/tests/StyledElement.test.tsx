import { StyledElement } from "../index";
import { render } from "@testing-library/react";
import React from "react";

describe("Test StyledElement", () => {
  test("StyledElement returns a renderable component", () => {
    const Component = StyledElement(
      "button",
      { color: "blue" },
      "",
      "afterburn"
    );

    const { container, unmount } = render(<Component />);

    expect(container).toMatchSnapshot();
    unmount();
  });
});
