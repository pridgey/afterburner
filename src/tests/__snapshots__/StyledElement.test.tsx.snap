// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test StyledElement StyledElement returns a renderable component 1`] = `
<div>
  <button
    class="afterburn"
  />
</div>
`;
